.storybook/test-runner.ts

```diff
@@ -1,7 +1,25 @@
-import type { TestRunnerConfig } from '@storybook/test-runner'
+import { waitForPageReady, type TestRunnerConfig } from '@storybook/test-runner'
+import { toMatchImageSnapshot } from 'jest-image-snapshot'
 
 const config: TestRunnerConfig = {
-  //
+  setup() {
+    expect.extend({ toMatchImageSnapshot })
+  },
+  async postVisit(page, context) {
+    await waitForPageReady(page)
+    const image = await page.screenshot()
+    console.log(`${process.cwd()}/__image_snapshots__`)
```

Comment:
Is this console.log necessary?

We don't need console.log because the test runner automatically outputs logs, and it may create noise in the console.